lib/BenchShow/Common.hs

```diff
@@ -81,6 +87,28 @@ filterSanity label old new = do
         ++ "\nOriginal groups: " ++ show old
         ++ "\nNew groups: " ++ show new
 
+type CSV = [[String]]
+
+-- XXX This is ugly in performance but works for the time being.
```

Comment:
Why would that be the case?

Reply:
I'm not truly streaming. I'm not using folds or parsers which I believe is more cleaner and the pipeline is more likely to fuse resulting in better performance.